# HG changeset patch
# User Rik <rik@octave.org>
# Date 1459545251 25200
#      Fri Apr 01 14:14:11 2016 -0700
# Node ID 31823239207e898db6adf51e7f5b4867555b1ad3
# Parent  82092a17fa50cc3ed46e9801616ee0a46dd177b7
maint: Use built-in warning from errwarn.h rather than calling warning_with_id.

* ov-str-mat.cc (octave_char_matrix_str::resize): Call
warn_implicit_conversion rather than warning_with_id with custom string.

diff --git a/libinterp/octave-value/ov-str-mat.cc b/libinterp/octave-value/ov-str-mat.cc
--- a/libinterp/octave-value/ov-str-mat.cc
+++ b/libinterp/octave-value/ov-str-mat.cc
@@ -161,19 +161,17 @@ octave_char_matrix_str::resize (const di
 }
 
 #define CHAR_MATRIX_CONV(T, INIT, TNAME, FCN) \
   T retval INIT; \
  \
   if (! force_string_conv) \
     err_invalid_conversion ("string", TNAME); \
  \
-  warning_with_id ("Octave:str-to-num", \
-                   "implicit conversion from %s to %s", \
-                   "string", TNAME); \
+  warn_implicit_conversion ("Octave:str-to-num", "string", TNAME); \
  \
   retval = octave_char_matrix::FCN (); \
  \
   return retval
 
 double
 octave_char_matrix_str::double_value (bool force_string_conv) const
 {
